include/sundials/sundials_memory.h

```diff
@@ -97,64 +97,68 @@ struct _SUNMemoryHelper_Ops
  * SUNMemory object.
  * The SUNMemory returned will not own the ptr, therefore, it will not free
  * the ptr in Dealloc. */
-SUNDIALS_EXPORT SUNMemory SUNMemoryHelper_Alias(SUNMemory mem);
+SUNDIALS_EXPORT
+SUNMemory SUNMemoryHelper_Alias(SUNMemory mem);
 
 /* Creates a new SUNMemory object with ptr set to the user provided pointer
  * The SUNMemory returned will not own the ptr, therefore, it will not free
  * the ptr in Dealloc. */
-SUNDIALS_EXPORT SUNMemory SUNMemoryHelper_Wrap(void* ptr, SUNMemoryType mem_type);
+SUNDIALS_EXPORT
+SUNMemory SUNMemoryHelper_Wrap(void* ptr, SUNMemoryType mem_type);
 
 /*
  * Required SUNMemoryHelper operations.
  */
 
+SUNDIALS_EXPORT SUNErrCode
+SUNMemoryHelper_Alloc(SUNMemoryHelper, SUNMemory* memptr, size_t mem_size,
```

Comment:
```suggestion
SUNDIALS_EXPORT 
SUNErrCode SUNMemoryHelper_Alloc(SUNMemoryHelper, SUNMemory* memptr, size_t mem_size,
```
(to be consistent with the formatting in the rest of this PR)